projects/smoldot-browser-demo/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@substrate/smoldot-browser-demo",
+  "version": "1.0.0",
+  "main": "./dist/index.js",
+  "author": "Raoul Millais",
```

Comment:
And another small thing: Since it's open source, a Parity project and it will most likely have more contributors than one I'd generally prefer this syntax
```
  "author": "Parity Team <admin@parity.io>",
  "contributors": [
    "Raoul Millais<raoul@parity.io>"
  ],
```

Reply:
100%. This is just the syntax that comes out of yarn init